train.py

```diff
@@ -52,6 +59,17 @@ def main(opts):
         opts.output_path = str(get_increased_path(opts.output_path))
     pprint("Running model in", opts.output_path)
 
+    # check if auto adventv2 works
```

Comment:
I think you could just write:
`If opts.train.lambdas.advent.is_auto_adventv2: {...}`
instead of using the brackets, like it's done in the rest of the file, to keep the same formatting